File: ElasticsearchClient/src/app/shared/models/request/filters-model.ts
export class FiltersModel {
  holidayYear: string[];
  regionName: string[];
  weekNumber: string[];
  responsibleRevenueManager: string[];
  parkName: string[];
  accommTypeName: string[];
  accommBeds: string[];
  unitGradeName: string[];
  accommName: string[];
  keyPeriodName: string[];
  lessParkWeekOccupancy?: number;
  greaterParkWeekOccupancy?: number;

  constructor() {
    this.holidayYear = [null];
    this.regionName = [null];
    this.weekNumber = [null];
    this.responsibleRevenueManager = [null];
    this.parkName = [null];
    this.accommBeds = [null];
    this.accommName = [null];
    this.accommTypeName = [null];
    this.unitGradeName = [null];
    this.keyPeriodName = [null];
    this.lessParkWeekOccupancy = null;
    this.greaterParkWeekOccupancy = null;
  }
}

File: Common/Models/FiltersModel.cs
using System.Collections.Generic;

namespace Common.Models
{
    public class FiltersModel
    {
        public IEnumerable<string> HolidayYear { get; set; }
        public IEnumerable<string> RegionName { get; set; }
        public IEnumerable<string> WeekNumber { get; set; }
        public IEnumerable<string> ResponsibleRevenueManager { get; set; }
        public IEnumerable<string> ParkName { get; set; }
        public IEnumerable<string> AccommTypeName { get; set; }
        public IEnumerable<string> AccommBeds { get; set; }
        public IEnumerable<string> UnitGradeName { get; set; }
        public IEnumerable<string> AccommName { get; set; }
        public IEnumerable<string> KeyPeriodName { get; set; }
        public string LessParkWeekOccupancy { get; set; }
        public string GreaterParkWeekOccupancy { get; set; }
    }
}